README.md

```
@@ -67,6 +93,34 @@ In order to run, the operator needs a secret to be created by the admin as follo
 oc create secret generic case-management-creds --from-literal=username=<username> --from-literal=password=<password>
 ```
 
+#### Required RBAC permissions
+
+The operator's service account (`must-gather-operator`) requires cluster-level **get/list/watch** on `serviceaccounts` to validate that the service account referenced in the MustGather CR exists before creating the gather job. When deploying via OLM this rule must be present in the CSV's `clusterPermissions` section. When deploying with manifests it is included in `deploy/02_must-gather-operator.ClusterRole.yaml`.
+
+If you see the following error in operator logs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)